types.go

```diff
@@ -98,13 +95,13 @@ func hugeIntFromNative(i *big.Int) (C.duckdb_hugeint, error) {
 	q.DivMod(i, d, r)
 
 	if !q.IsInt64() {
-		return C.duckdb_hugeint{}, fmt.Errorf("big.Int(%s) is too big for HUGEINT", i.String())
+		return m.HugeInt{}, fmt.Errorf("big.Int(%s) is too big for HUGEINT", i.String())
 	}
 
-	return C.duckdb_hugeint{
-		lower: C.uint64_t(r.Uint64()),
-		upper: C.int64_t(q.Int64()),
-	}, nil
+	var hugeInt m.HugeInt
```

Comment:
Maybe initialize via `NewHugeInt(r.Uint64(), q.Int64())`?

Reply:
This would align with @james-lawrence's suggestion. Alternatively, we *could* expose the fields of the struct type directly. Wdyt?

Reply:
I think exposing the fields would be ideal, no harm in that right? What is the difference if we have getters and setters for both anyways

Reply:
Tbh I am not even sure how to expose the fields if we alias the struct itself.
If that's possible, then I am happy to do it.

Currently, the bindings package looks like this:
```Go
type (
	Date              = C.duckdb_date
)
```
And the C header, for example, for `duckdb_date`:
```c
typedef struct {
	int32_t days;
} duckdb_date;
```